src/module/common/button.tsx

```diff
@@ -1,28 +1,109 @@
-import { cva, type VariantProps } from "class-variance-authority";
-import React from "react";
+import { cva, type VariantProps } from 'class-variance-authority'
+import React from 'react'
 
 export interface ButtonVariants extends VariantProps<typeof buttonVariants> {}
-const buttonVariants = cva(["btn"], {
-  variants: {
-    intent: {
-      default: [],
-      outline: ["btn-outline"],
+const buttonVariants = cva(
+  ['btn', 'overflow-hidden', 'relative', 'rounded', 'border','transition-all','duration-500'],
+  {
+    variants: {
+      variant: {
+        primary: 'bg-btn-blue',
+        secondary: 'transparent',
+      },
+      size: {
+        medium: 'px-6 py-2.5 w-400 mb-5',
+      },
+      animation: {
+        'login-gradient': `
```

Comment:
being somewhat picky here => maybe we can call this animation with something more generic, maybe `growing-bubble-tl`?

also can you make this a regular string instead of template string? hahah 

one other thought would be moving this to `tailwind.config.js`, but it's probably fine where it is right now.

Reply:
What's `tl` for? top to the left?
Then what suggestion do you have for the `register-gradient`?

Reply:
Hmmm maybe use `primary` or `secondary` to distinguish them for now?  

Or ask ChatGPT for suggestions. 🤪